update fundamental styles library version in angular example 0.8.1 (#1245)
SAP/luigi: website/fiddle/README.md

[![Netlify Status](https://api.netlify.com/api/v1/badges/6e36180d-2486-4daf-80a5-90a0ab363992/deploy-status)](https://app.netlify.com/sites/relaxed-kowalevski-49f468/deploys)

> Luigi fiddle, deployed via netlify and accessible at https://fiddle.luigi-project.io
> Go to the [Luigi Fiddle](https://fiddle.luigi-project.io) site to see Luigi in action and configure a sample application.
> 
> ### Code formatting for contributors

Reviewer: ```suggestion
Luigi Fiddle, deployed via Netlify and accessible at https://fiddle.luigi-project.io.
The Fiddle allows you to see Luigi in action and configure a sample application.

### Install Luigi Fiddle on your machine
```

Just a small comment on the readme 🙂
